lib/rubocop/cop/mixin/preferred_delimiters.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    # Common functionality for handling percent literal delimiters.
+    class PreferredDelimiters
```

Comment:
I think this class name is a bit too generic, since this class deals exclusively with percent literal delimiters, but there are a lot of delimiters in Ruby. 🙂